ui/app/adapters/application.js

```diff
@@ -41,15 +41,15 @@ export default RESTAdapter.extend({
       headers['X-Vault-Wrap-TTL'] = options.wrapTTL;
     }
     let namespace = typeof options.namespace === 'undefined' ? this.namespaceService.path : options.namespace;
-    if (namespace && !NAMESPACE_ROOT_URLS.some(str => url.includes(str))) {
+    if (namespace && !NAMESPACE_ROOT_URLS.some((str) => url.includes(str))) {
```

Comment:
assuming this is a new linting rule to use parens in a single line arrow function?